OpenAI-DotNet-Tests/TestFixture_09_FineTuning.cs

```diff
@@ -201,16 +202,13 @@ public async Task Test_08_DeleteFineTunedModel()
             Assert.IsNotNull(models);
             Assert.IsNotEmpty(models);
 
-            foreach (var model in models)
+            foreach (var model in models.Where(x => x.OwnedBy.StartsWith("user-")))
```

Comment:
This could be org- or user-

Reply:
Do you want me to re-submit the PR? I guess it could be changed to any model where the owner is not "openai" or "system"

Reply:
It's fine, I'll do it here in a second.

Reply:
@ADefWebserver are are not setting `OrganizationId`, correct?

Reply:
I am setting OrganizationId (bcause even though I have a Personal account, I have an OrganizationId) following the directions from this Blog post: https://blazorhelpwebsite.com/ViewBlogPost/2066